packages/core/useVModel/index.ts

```diff
@@ -70,8 +70,10 @@ export function useVModel<P extends object, K extends keyof P, Name extends stri
 
   event = eventName || event || `update:${key}`
 
+  const getValue = () => isDef(props[key!]) ? props[key!] : defaultValue
```

Comment:
Or we can determine if the property exists in the object and thus determine if the default value is obtained

```ts
  const getValue = () => Object.hasOwnProperty.call(props, key!) ? props[key!] : defaultValue
```